src/js/components/purchaseOrder/PurchaseOrderListFilters.jsx

```diff
@@ -170,12 +162,12 @@ const PurchaseOrderListFilters = ({
       fetchStatuses();
     }
 
-    if (!buyers || buyers.length === 0) {
+    if (!buyers) {
```

Comment:
I changed initialValue of `buyers` to `null`, because having it as `[]` was a bad idea, because if we had situation, when we would fetch none organization and made the `if` above like this: `buyers.length > 0` we would actually never fetch if we don't have buyers' organizations (even if we don't actually need them (we don't have central purchasing enabled).